content/docs/start/experiments-trail/experiments-1.md

```diff
@@ -0,0 +1,334 @@
+> rest of your system. This prevents version conflicts.
+
+```dvc
+$ git clone https://github.com/iterative/get-started-experiments -b source-code
+$ cd get-started-experiments
+$ virtualenv .venv
+$ . .venv/bin/activate
+$ python -m pip install -r requirements.txt
+```
+
+### Get the data set
+
+The repository you cloned doesn't contain the dataset. In order to get
+`fashion-mnist.tar.gz` from `dataset-registry`, we use `dvc get` to download the
+missing data files. `dvc get` is similar to `wget` but works with Git+DVC
+repositories to download binary files.
+
+```dvc
+$ dvc get https://github.com/iterative/dataset-registry \
+          fashion-mnist/images.tar.gz -o data/images.tar.gz
+```
+
+Then we extract this file that contains labeled images.
+
+```dvc
+$ tar -xvzf data/images.tar.gz --directory data/
+```
+
+### Specify the experiment
+
+We first initialize DVC inside the project to create an experiment.
+
+```dvc
+$ dvc init
+```
+
+Then we add the dataset to the project:
+
+```dvc
+$ dvc add data/images
+```
+
+This creates a `data/images.dvc` file that contains all the relevant metadata of
+the directory. You can add and commit `data/images.dvc` file, and changes in
+`.gitignore` that hides the `data/images/` directory from Git.
+
+```dvc
+$ git add data/.gitignore data/images.dvc
+$ git commit -m "Dataset added"
+```
+
+DVC experiments are run by specifying their commands, outputs, parameters and
+dependencies. We specify an experiment command by `dvc stage add`.
+
+```dvc
+$ dvc stage add -n train \
+                -p model.conv_units \
+                -p train.epochs \
+                -d data/images \
+                -m metrics.json \
+                python3 src/train.py
+```
+
+The command tells DVC to create an experiment named `train`, and that for any
+change in `data/images/`, `model.conv_units` or `train.epochs`, we run an
+experiment using `src/train.py` that produces a new `metrics.json` file.
+
+DVC creates `dvc.yaml` file and modifies `.gitignore` for Git to ignore certain
+artifacts.
+
+```dvc
+$ git add dvc.yaml .gitignore
+$ git commit -m "added data and the experiment"
+```
+
+DVC is ready to run the experiments now!
+
+</details>
+
+## Running the experiment with default parameters
+
+The purpose of `dvc exp` subcommands is to run the pipeline for ephemeral
+experiments. By _ephemeral_ we mean the experiments can be run without
+committing parameter and dependency changes to Git. Instead the artifacts
+produced for each experiment are tracked by DVC and persisted on demand.
+
+Running the experiment with default hyperparameter values requires only the
+command:
+
+```dvc
+$ dvc exp run
+...
+Reproduced experiment(s): exp-7683f
+Experiment results have been applied to your workspace.
+
+To promote an experiment to a Git branch run:
+
+        dvc exp branch <exp>
+```
+
+It runs the pipeline starting from the basic dependencies and produces
+`metrics.json` file for the default state.
```

Comment:
It feels like we're assuming the user will read the long `<details>` preparation section. That's kind of optional reading, I think? For those who want to actually try the sample commands. I speculate that most people only want to read the page and get an idea of the feature so you may have to rethink how to explain this part (if at all).

Reply:
Same applies in some other explanations throughout the text.

Reply:
I see your point and reworded a bit but assuming the reader _did not_ read the installation section may result in too verbose or abstract explanations. I rather err on the side of being concrete.

Reply:
Thanks, it's good to be concrete/ specific in these cases. But

> It runs the specified command

What command? Sounds like you mean `dvc exp run`.

I guess this sentence is a bit repetitive since the one before the code block already stated what we're doing. You could just go for "You should now wee metrics.json in your workspace." if that's the point of this sentence.